aksel.nav.no/website/components/layout/god-praksis-page/GodPraksisPage.tsx

```diff
@@ -0,0 +1,43 @@
+import dynamic from "next/dynamic";
+import { Box, Page, VStack } from "@navikt/ds-react";
+import Footer from "@/layout/footer/Footer";
+import { GpEntryPageProps } from "@/layout/god-praksis-page/types";
+import useInitialState from "@/layout/god-praksis-page/useInitialState";
+import Header from "@/layout/header/Header";
+import Hero from "./hero/Hero";
+
+const ArticleList = dynamic(() => import("./articles/ArticleList"), {
```

Comment:
Ikke stor fan av at vi må gjøre dette for å bruke queries riktig, men ikke klar over en bedre metode akkurat nå.